### PR TITLE
Add Razor2::Client::Agent perl module

### PR DESCRIPTION
**Pull request by fritzkink**

### CODE REVIEW

File: components/perl/Razor2-Client-Agent/Makefile [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+#   $WS_TOOLS/perl-integrate-module Razor2::Client::Agent
+#
+
+BUILD_STYLE = makemaker
+
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_PERL_MODULE =		Razor2::Client::Agent
+HUMAN_VERSION =			2.86
+COMPONENT_REVISION =		1
```
Comment (mtelka) [inline components/perl/Razor2-Client-Agent/Makefile:23]: This line should not be there.  I'm curious why it appeared here.  Did you run  the `perl-integrate-module` script twice and then squashed both changesets?  Or maybe there is a bug in the `perl-integrate-module` script...

Anyway, this is harmless, so no problem to leave it here.

**Review by mtelka — APPROVED** [review]
This apparently replaces #9062, so #9062 could be closed.

File: components/perl/Razor2-Client-Agent/Makefile (hunk 1 above)
Comment (AndWac) [reply components/perl/Razor2-Client-Agent/Makefile:23]: Shall I merge this as is or wait for a correction?

Comment (mtelka) [reply components/perl/Razor2-Client-Agent/Makefile:23]: > Shall I merge this as is or wait for a correction?

Correction is not needed, but I'd like to know the answer to my question.
So feel free to merge.

Thanks.

**Pull request merged**